<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="5" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="6" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="maximumSize">
           <size>
            <width>1920</width>
            <height>1080</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1327</width>
             <height>69</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>851</width>
              <height>486</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="topMargin">
              <number>30</number>
             </property>
             <property name="spacing">
              <number>1</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="iconSize">
           <size>
            <width>500</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="undo">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color:rgb(225, 225, 225);
border-style: solid;
border-width: 5px;
border-radius: 10px;


}

QPushButton::hover{
background-color:rgb(225, 225, 225,100);
border-style: solid;
border-width: 5px;
border-radius: 10px;


}



</string>
          </property>
          <property name="text">
           <string>Annuler</string>
          </property>
          <property name="icon">
           <iconset resource="img.qrc">
            <normaloff>:/img/undo.png</normaloff>:/img/undo.png</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <layout class="QGridLayout" name="gridTest"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_eval">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Évaluez la différence entre les deux profils :</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="valider">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>VALIDER L'ORDRE</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item row="9" column="0">
         <layout class="QHBoxLayout" name="gridRank"/>
        </item>
        <item row="15" column="0">
         <widget class="QPushButton" name="save">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SAUVEGARDER LES RESULTATS</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QPushButton" name="suite">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>AFFICHER LES RESULTATS</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridCompa">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item row="16" column="0">
         <widget class="QPushButton" name="quit">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>QUITTER</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="historique">
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>1000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Historique :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Réorganisez les profils du plus fragile au moins fragile. (Clic droit sur un profil pour l'agrandir)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelValeur">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QLabel" name="text_1">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>1061</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cette application vous propose de classer 20 profils de personnes âgées évalués selon quatre critères, respectivement les états 
physique, psychosocial, nutritionnel et cognitif de la personne.</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>1100</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Les profils sont proposés dans un ordre aléatoire, c'est à vous de les réorganiser du plus fragile au moins fragile. 
Il n'est pas possible de placer un profil après un autre qui lui est plus fragile en tout point. </string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>280</y>
          <width>1071</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vous observerez que certains profils ont une couleur différente : ce sont des profils &quot;palliers&quot; qui servent à faciliter le classement. 

Pour modifier le classement d'un profil, cliquez dessus et faites le glisser sur le profil à droite de l'emplacement désiré. 
Une fois le classement validé, il vous faudra choisir l'intensité de votre différence de préférence entre deux profils consécutifs.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cont">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>450</y>
          <width>330</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Continuer</string>
        </property>
       </widget>
       <widget class="QToolButton" name="exProfil">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>181</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>180</width>
          <height>115</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="labelExProfil">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Exemple de profil</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>80</y>
          <width>831</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="img.qrc">:/img/icons.PNG</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>210</y>
          <width>61</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="img.qrc">:/img/jauge.PNG</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>460</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Jauge de fragilité : plus la barre est
 haute, plus le sujet est fragile 
sur le critère associé.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QPushButton" name="begin">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>190</y>
          <width>241</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Commencer</string>
        </property>
       </widget>
       <widget class="QLabel" name="title">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>30</y>
          <width>651</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>20</pointsize>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Projet KAMI : Élicitation d'un modèle de préférences</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>140</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Charger des données</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QLabel" name="text_6">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>1181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A présent, vous allez comparer deux à deux les profils que vous venez d'ordonner. 
Vous avez le choix entre 4 catégories sémantiques de différence de fragilité : 
aucune différence (0)  - différence faible (1) - différence intermédiaire (2) - différence forte (3).</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="cont2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>300</y>
          <width>330</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Continuer...</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionQuitter"/>
    <addaction name="actionInstructions"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Réiniialiser</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
